core-bundle/src/DependencyInjection/Security/ContaoLoginFactory.php

```diff
@@ -12,40 +12,30 @@
 
 namespace Contao\CoreBundle\DependencyInjection\Security;
 
-use Symfony\Bundle\SecurityBundle\DependencyInjection\Security\Factory\FormLoginFactory;
+use Symfony\Bundle\SecurityBundle\DependencyInjection\Security\Factory\AbstractFactory;
 use Symfony\Component\DependencyInjection\ChildDefinition;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Reference;
 
-class ContaoLoginFactory extends FormLoginFactory
+class ContaoLoginFactory extends AbstractFactory
 {
     public function __construct()
     {
-        parent::__construct();
-
-        $this->addOption('lock_period', 300);
-        $this->addOption('login_attempts', 3);
+        $this->options = ['require_previous_session' => false];
```

Comment:
This unsets the other two options `check_path` and `use_forward`. Is this intended?

Reply:
Yes, they will be gone in #1130 anyway. We also unset any other options, as they do not make sense for anything in Contao.